### DISCUSSION TITLE
OBS stream to Liquidsoap

### DISCUSSION THREAD

**Discussion by sorintodys** [savonet/liquidsoap#2144] [Q&A] [answered]
Hello,

Please let me know if any one experience streaming with obs to liquidsoap.

Thank you!

**Comment by toots** [comment, marked as answer]
Yes! This has been discussed and debugged as part of the `2.0.2` debug release but on slack, unfortunately.

A typical script looks like this:
```ruby
s = input.rtmp("rtmp://localhost:1935/live")

output.srt(s, fallible=true, %ffmpeg(format="mpegts", %audio.copy, %video.copy), ...)
```

**Comment by sorintodys** [comment]
Hello,
Thank you very much for answer. Liquidsoap is nice application, but after video and ffmpeg is become crazy hottttttttt :)))
